backend-project/config/settings/base.py

```diff
@@ -147,3 +147,5 @@
 MINIO_ACCESS_KEY = env("MINIO_ACCESS_KEY")
 MINIO_SECRET_KEY = env("MINIO_SECRET_KEY")
 MINIO_URL = env("MINIO_URL")
+
+TEST_RUNNER = "config.tests.CustomDiscoverRunner"
```

Comment:
Usuń to.

Reply:
W innym wypadku musimy przekazywać pattern do komendy django. Chyba że jest inne wyjście o którym nie wiem. Można też iść w kierunku pytest, widziałem bibliotekę pytest django.

Reply:
W innym wypadku musimy przekazywać pattern do komendy django. Chyba że jest inne wyjście o którym nie wiem. Można też iść w kierunku pytest, widziałem bibliotekę pytest django.